protostar/compiler/project_compiler.py

```diff
@@ -37,6 +45,11 @@ class ProjectCompilerConfig:
     hint_validation_disabled: bool = False
 
 
+class OutputType(Enum):
```

Comment:
I don't understand why is that enum used in the compiler, do we need it outside the `ProjectCompiler` class?

Reply:
It's not needed outside, I just think using enum here provides stronger typing than relying on strings or other solution. Prefixed it with underscore so it's obvious it's internal.